test_typeddict_errors fails on python3.11

Heads up, we're seeing a test failure with python 3.11.0rc2 and mypy_extensions. Python 3.11 will be released in October.

```
======================================================================
FAIL: test_typeddict_errors (testextensions.TypedDictTests.test_typeddict_errors)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/source/tests/testextensions.py", line 91, in test_typeddict_errors
    with self.assertRaises(TypeError):
AssertionError: TypeError not raised

----------------------------------------------------------------------
Ran 7 tests in 0.003s

FAILED (failures=1)
```

--- a/tests/testextensions.py
+++ b/tests/testextensions.py
@@ -89,9 +89,9 @@ class TypedDictTests(BaseTestCase):
         with self.assertRaises(TypeError):
             issubclass(dict, Emp)  # type: ignore
         with self.assertRaises(TypeError):
-            TypedDict('Hi', x=1)
+            TypedDict('Hi', x=())
         with self.assertRaises(TypeError):
-            TypedDict('Hi', [('x', int), ('y', 1)])
+            TypedDict('Hi', [('x', int), ('y', ())])
         with self.assertRaises(TypeError):
             TypedDict('Hi', [('x', int)], y=int)
 

